sovrin_client/test/cli/test_merge_invitation.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from sovrin_client.agent.agent_issuer import AgentIssuer
+from sovrin_client.agent.agent_prover import AgentProver
+from sovrin_client.agent.agent_verifier import AgentVerifier
+from sovrin_client.agent.walleted import Walleted
+from sovrin_client.client.wallet.connection import Connection
+from sovrin_client.client.wallet.wallet import Wallet
+from sovrin_client.agent.walleted_agent import WalletedAgent
+from plenum.common.constants import NAME, VERSION, ATTRIBUTES, VERIFIABLE_ATTRIBUTES
+
+
+def test_merge_invitation():
```

Comment:
The test checks merging to an empty link only. Can we also check that merging works for non-empty link (real merging?)